picsur/docker-compose.yml

```
@@ -0,0 +1,44 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: picsur_web_1
+      APP_PORT: 8286

```

Proxy auth not needed for this app.
```suggestion
      APP_PORT: 8286
      PROXY_AUTH_ADD: "false"
```